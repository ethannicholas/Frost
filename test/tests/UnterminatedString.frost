method main() {
    Console.printLine("Hello, World!)
}
