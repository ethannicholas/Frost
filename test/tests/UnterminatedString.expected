UnterminatedString.panda:2:23: error: unterminated string literal
1 error
